cmd/generate/config/rules/meraki.go

```diff
@@ -0,0 +1,26 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func Meraki() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Cisco Meraki is a cloud-managed IT solution that provides networking, security, and device management through an easy-to-use interface.",
+		RuleID:      "meraki-api-key",
+		Regex: utils.GenerateSemiGenericRegex([]string{"meraki"},
+			`[0-9a-f]{40}`, false),
+		Keywords: []string{"meraki", "cisco_meraki"},
+	}
+
+	// validate
+	tps := utils.GenerateSampleSecrets("meraki", secrets.NewSecret(utils.Hex("40")))
+	fps := []string{
+		`│e9e0f062f587b_423bb6cc6328eb786d75b45783e.bfu`,
+		`a1cdda32f587g423bb6cc6328Eb786d75b45783e`,
+	}
```

Comment:
We should strive to add realistic false-positives for each rule. This can include:

- low entropy (`xxxxxxxxxxx`)
- prefix overlaps  (`  ReactNativeCameraKit: f15a5a04b0f6dc6073e6db0296e6ef2d8b8d2522`, see #1491)
- invalid case (`acdeF...`)
- invalid characters( `abdefghj...`)
- invalid length (`meraki_token = 5cb4a5f04cd412fe946667b17f0129ba17aeb2e0c7b5b7264efcebf7d022bfe2R21`)